### PR TITLE
Change stylesheets for radio button with icons in form wizard. Closes #218

### LINKED ISSUE #218: Form type radio styles change
1. Align radio vertically with step index circle http://snpy.in/tqUX5h.
2. Change fill color of `.radioButton input:not(:checked) ~ .radioButton__content .radioButton__iconContent .icon path ` to `fill: #5F6871;`
3. Add hover classes for radio
```
.radioButton input:not(:checked) ~ .radioButton__content:hover .radioButton__iconContent .icon path:first-child {
    opacity: .4;
}    
```
and
```
.radioButton input:not(:checked) ~ .radioButton__content:hover .radioButton__iconContent .icon path {
    opacity: .8;
}
```

### PR DESCRIPTION
**Pull request by astupak**
https://github.com/sendsay-ru/account/issues/218

### CODE REVIEW

**Comment by zouxuoz** [comment]
lgtm 🔥

**Comment by radiansz** [comment]
I think color of radios should be same at second and forth step

File: src/blocks/radioButton/radioButton.less [hunk 1]
```diff
@@ -216,12 +220,19 @@
     cursor: default;
 }
 
-.radioButton  input:not(:checked) ~ .radioButton__content .radioButton__iconContent .icon path{
-   fill: #888;
+.radioButton  input:not(:checked) ~ .radioButton__content .radioButton__iconContent .icon_formType path{
+   fill: #5F6871;
```
Comment (radiansz) [inline src/blocks/radioButton/radioButton.less]: Move this color to a variable

**Comment by trikashny** [comment]
LGTM

**Pull request merged**